sdk/state/ingest.go

```diff
@@ -15,71 +15,67 @@ import (
 // determine if successful or not, and understand changes in the ledger as a
 // result.
 func (c *Channel) IngestTx(tx *txnbuild.Transaction) error {
-	// Use the tx to update the latest unauthorized close agreement if possible.
-	err := c.ingestTxToUpdateUnauthorizedCloseAgreement(tx)
+	// TODO: Use the transaction result to affect on success/failure.
+
+	err := c.ingestTxToUpdateCloseState(tx)
```

Comment:
Could we keep `c.ingestTxToUpdateUnauthorizedCloseAgreement` and `c.ingestTxToUpdateCloseState` as separate functions? They appear to update different things and we probably don't gain a lot by coupling them together. We can merge them later or abstract shared state if it looks like it would be beneficial.

Reply:
separated here
https://github.com/stellar/experimental-payment-channels/pull/232/commits/bc6c6bc6c1ed02e3b54f7fc2f4a29a3c72d22f5b